### DISCUSSION TITLE
Form field and label width.

### DISCUSSION THREAD

**Discussion by rhysflook** [filamentphp/filament#11155] [Help] [answered]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

This seems like something that should be pretty simple but for the life of me a can't find anything in the documentation.

I just want to adjust the width of form fields so that they take up more of the page, without being moved to the left, like in the attached picture. (Achieved with a group of ugly css selectors)

![image](https://github.com/filamentphp/filament/assets/53523326/ff2091f4-9d6b-4fda-b161-823928ed8156)

I had tried to get the layout I want using the Grid class which is fine for my other pages but the fields need to be wider on this one.

**Comment by leandrocfe** [comment, marked as answer]
You can use Group/columns/columnSpan to achieve this
```php
Forms\Components\Group::make([
    Forms\Components\Placeholder::make('Name'),
    Forms\Components\TextInput::make('name')
        ->hiddenLabel()
        ->columnSpan(11),
])
    ->columns(12)
    ->columnSpanFull()
```

**Reply by rhysflook** [reply]
This is much better than my funky css, thanks!
